### ISSUE TITLE
Integrate with a reference manager?

### ISSUE THREAD

**Issue by young24** [young24/Perfect-Lab-System#1]
I just searched the market of the reference managers. I don't like EndNote. There are 3 plausible options:
Zotero, ReadCube, and Mendeley.
### Pros
- Free
- Open Source
- link with NutStore with WebDev

### Cons
- only 300 MB storage
- Chrome Extension works poorly with some journals. (e.g., cell)
- only support PDFs

### Pros
- good web interface

### Cons
- Not free

### Pros
- Large Free Storage (2 GB)
- good Chrome plug-in
- good web-interface

### Cons
- not sure whether it can link with cloud services

**Comment by young24** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [integrate the Reference folder with reference managers](https://trello.com/c/Dm78yxXl/32-integrate-the-reference-folder-with-reference-managers)

**Comment by xutq** [comment]
Mendeley's build-in search engine is wired.
 
![image](https://user-images.githubusercontent.com/15274805/43509086-37b32c06-95a4-11e8-9a54-abde2d56c824.png)

**Comment by young24** [comment]
It is perfect on my computer, however, I believe you can adjust it manually. @xutq

**Comment by young24** [comment]
I'm just aware that we can just add a link to the web library in a markdown to achieve the goal. So that, 

1. click the link
2. jump to the web library
3. Done.

Then if you want to download any files from the library, just click on the download button of that paper in the web library. (This can be achieved easily by Mendeley.)
@xutq

**Comment by xutq** [comment]
Should this library be set to a shared library?

**Comment by young24** [comment]
Yes, you have to invite other team members to share this library. Moreover, Mendeley provides links for each file in the library, which allows us to manipulate files separately.  
So that, actually, we can build a hyperlink-style library on GitHub with the help of Mendeley. It is approximately my ideal storage model. Put real files in the Mendeley library and put their links on GitHub. @xutq

**Comment by xutq** [comment]
What is the situation of tracking the changes of references?

**Comment by young24** [comment]
When you add/remove/edit an item to/from/in the library @xutq

**Comment by xutq** [comment]
What is the reason/necessity to track them?
